RELEASES.md

```diff
@@ -1,3 +1,101 @@
+Version 1.67.0 (2023-01-26)
+==========================
+
+<a id="1.67.0-Language"></a>
```

Comment:
These new anchors (https://github.com/rust-lang/relnotes/pull/111) do work, but they don't change the link you get when you hover over the section header. Not sure if that's expected -- cc @pnkfelix.

Reply:
AFAIK, changing the link isn't possible with GitHub flavored markdown.